Libraries/Text/TextInput/RCTBaseTextInputViewManager.m

```diff
@@ -58,6 +58,7 @@ @implementation RCTBaseTextInputViewManager
 RCT_EXPORT_VIEW_PROPERTY(blurOnSubmit, BOOL)
 RCT_EXPORT_NOT_OSX_VIEW_PROPERTY(clearTextOnFocus, BOOL) // TODO(macOS GH#774)
 RCT_REMAP_NOT_OSX_VIEW_PROPERTY(keyboardType, backedTextInputView.keyboardType, UIKeyboardType) // TODO(macOS GH#774)
+RCT_EXPORT_NOT_OSX_VIEW_PROPERTY(showSoftInputOnFocus, BOOL) // TODO(macOS GH#774)
```

Comment:
I don't see this in our repo. Why does it have the macOS GH tag?

Reply:
My understanding has been to add the GH tag whenever we introduce a divergence from upstream. Is my understanding correct, or is there something more to this?

Reply:
That's spot on! Why do we need this divergence?

Reply:
From what I've been able to tell, `showSoftInputOnFocus` seems to be a mobile-specific thing. It indicates whether or not the virtual keyboard appears when we focus on a text input, which doesn't seem to make sense on macOS as far as I'm aware.